xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.cpp

```diff
@@ -21,43 +21,91 @@
 #include "RendererAML.h"
```

Comment:
I removed the inheritance to LinuxRenderGLES because after some mails with AMLogic it will never be a solution and will never be targetted to use GPU for rendering. GPU will never be working with 4K and DRM.